src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = (props) => {
```

Comment:
[OPTIONAL] ♻
Consider using spread operator for the props object.

Reply:
Hi, Reshy:wave:
Thanks for your review:pray:
How do I use spread operator here, please?
Do you mean using object destructuring for the props?

Reply:
Yes, instead of having a single object and then refering to each by using `props.name` , you can use destructuring like `const InputTodo = { addTodoProps } `  and then refer to that function by simply calling it, instead of using `props.addTodoProps` .
I put it as optional change but it's usually recommended to use by linter checks.